.gitignore

```diff
@@ -23,6 +22,8 @@ tests/behat.yml
 # Contrib
 modules/contrib/*
 themes/contrib/*
+core
```

Comment:
This .gitignore sits inside /profiles/df

so there should never be a core or a profiles folder inside of the DF profile.